addons/misc/CfgVehicles.hpp

```diff
@@ -433,6 +433,546 @@ class CfgVehicles {
                     showDisabled = 0;
                     icon = QPATHTOF(ui\Bubble_Wrap_Item.paa);
                 };
+
+                class Unpack_kat_IFAK_Item {
+                    displayName = CSTRING(IFAK_Unpack);
+                    condition = QUOTE([ARR_3(_player, ""kat_IFAK"", 0)] call FUNC(checkFAKSlot));
+                    exceptions[] = {"isNotSwimming", "isNotInside", "notOnMap", "isNotSitting"};
+                    statement = QUOTE([ARR_3(_player, ""kat_IFAK"", 0)] call FUNC(unpackFAK));
+                    showDisabled = 0;
+                    icon = QPATHTOF(ui\IFAK.paa);
+
+                    class Slot1 {
+                        displayName = CSTRING(FAK_Slot_1);
+                        condition = QUOTE([ARR_3(_player, ""kat_IFAK"", 1)] call FUNC(checkFAKSlot));
+                        exceptions[] = {"isNotSwimming", "isNotInside", "notOnMap", "isNotSitting"};
+                        statement = QUOTE([ARR_3(_player, ""kat_IFAK"", 1)] call FUNC(unpackFAK));
+                        showDisabled = 0;
+                        icon = QPATHTOF(ui\IFAK.paa);
+                    };
+
+                    class Slot2 {
+                        displayName = CSTRING(FAK_Slot_2);
+                        condition = QUOTE([ARR_3(_player, ""kat_IFAK"", 2)] call FUNC(checkFAKSlot));
+                        exceptions[] = {"isNotSwimming", "isNotInside", "notOnMap", "isNotSitting"};
+                        statement = QUOTE([ARR_3(_player, ""kat_IFAK"", 2)] call FUNC(unpackFAK));
+                        showDisabled = 0;
+                        icon = QPATHTOF(ui\IFAK.paa);
+                    };
```

Comment:
Inherit entries to avoid copy pasting
```suggestion
                    class Slot2: Slot1 {
                        displayName = CSTRING(FAK_Slot_2);
                        condition = QUOTE([ARR_3(_player, 'kat_IFAK', 2)] call FUNC(checkFAKSlot));
                        statement = QUOTE([ARR_3(_player, 'kat_IFAK', 2)] call FUNC(unpackFAK));
                    };
```